testbed/tests/test_fonts.py

```diff
@@ -76,11 +76,11 @@ async def test_font_options(widget: toga.Label, font_probe):
 @pytest.mark.parametrize(
     "font_family,font_path,font_kwargs",
     [
-        # OpenType font, no options
+        # OpenType font with weight property
         (
             "Font Awesome 5 Free Solid",
             "resources/fonts/Font Awesome 5 Free-Solid-900.otf",
-            {},
+            {"weight": BOLD},
```

Comment:
This font is internally marked as bold (as indicated by the "900" in the name), and Android reports it as such.